services/src/datasets/external/netcdfcf/mod.rs

```diff
@@ -42,15 +47,19 @@ use geoengine_operators::{
 };
 use log::debug;
 use serde::{Deserialize, Serialize};
-use snafu::ensure;
+use serde_json::json;
 use snafu::{OptionExt, ResultExt};
 use std::collections::VecDeque;
 use std::io::BufReader;
 use std::path::{Path, PathBuf};
 use std::str::FromStr;
 use walkdir::{DirEntry, WalkDir};
 
-mod error;
+#[cfg(feature = "ebv")]
+mod ebvportal_api;
+#[cfg(feature = "ebv")]
+mod ebvportal_provider;
```

Comment:
Do we need all the cfgs? I thought this whole module only gets inserted if `ebv` is activated.

Reply:
the netcdfdf mod has no conditional compiliation flag

Reply:
If you rely on that the entity is always called ebv_cube, it is not a generic NetCDF provider, right?

Reply:
do you mean NetCDF_CF_? Idk actually. But what has this to do with the compilation flags?

Reply:
It would make sense to have a generic NetCDF-CF provider.
However, relying on the EBV standard (EBV 1.0) means we always have entities and they are always encoded in the array `ebv_cube`. This is not generic, so I would pack it under the `ebv` flag.

Reply:
e42fe9a87e42e94869a4a247f12f0baf073e80ae